apps/challenges/models.py

```diff
@@ -131,6 +131,10 @@ def __init__(self, *args, **kwargs):
     task_def_arn = models.CharField(
         null=True, blank=True, max_length=2048, default=""
     )
+    # The service name for the challenge which contains all the tasks
```

Comment:
Change `The service name for the challenge which contains all the tasks` -> `Challenge evaluation worker service name`